style.css

```diff
@@ -0,0 +1,604 @@
+		padding: 0 0 0 14px;		
+	}
+
+	.goods__item:nth-last-child(3) {display: none;}
+
+	.goods__img {
+		height: 165px;
+		padding:0 50px 0;
+	}
+
+	.goods__name {
+		font-size: 15px;
+		margin: 0;
+	}
+
+	.goods__details {
+		height: 32px;
+		font-size: 20px;
+	}
+
+	.goods__color, .goods__color_active {
+		width: 15px;
+		height: 15px;
+		margin-right: 5px;
+	}
+}
+
+/*MAIN <section class='brand-info'>*/
+.brand-info {
+	margin: 0 0 79px 0;
+	height: 494px;
+	background: #000000;
+	padding: 67px 45px 66px 33px;
+	display: flex;
+}
+
+.brand-info__image {
+	border: 8px;
+	display: block;
+	flex: 1 1 70%;
+	margin: 0 42px 0 0;
+}
+
+.brand-info__text{
+	color: #FFFFFF;
+	flex: 0 1 30%;
+	display: flex;
+	flex-direction: column;
+	padding: 87px 0 0 0;
+}
+
+.brand-info__history {
+	margin: 0 0 22px 0;
+	line-height: 17px;
+}
+
+.brand-info__title, .brand-info__history {text-align: left;}
+
+.brand-info__title {line-height: 27px;}
+
+.brand-info__link {
+	display: block;
+	color: #FFFFFF;
+	font-size: 12px;
+}
+
+.brand-info__link::after {
+	content: url('Image_Axion/Vector/Arrow (1).png');
+	display: inline-block;
+	width: 42.83px;
+	margin-left: 8.91px;
+}
+
+@media (max-width: 768px) {
+	.brand-info {
+		position: relative;
+		flex-direction: column-reverse;
+		padding: 0 40px 100px 40px;
+		height: 650px;
+	}
+
+	.brand-info:before {
+	  content: "";
+	  background: #000000;
+	  transform: rotate(2.6deg);
+	  position: absolute;
+	  width: 200%;
+	  height: 100%;
+	  left: -20%;
+	  top: -7%;
+	  z-index: -1;
+	}
+
+	.brand-info__image {
+		margin: 0;
+		height: 370px;
+		flex: 0 1 auto;
+	}
+
+	.brand-info__text {
+		padding: 0;
+		margin: 20px 0 20px 0;
+		flex: 0 1 auto;
+		align-items: center;
+	}
+
+	.brand-info__title {
+		line-height: 27px;
+		text-align: center;
+}
+
+	.brand-info__history {
+		text-align: center;
+		font-size: 20px;
+		line-height: 25px;
+	}
+
+	.brand-info__link {display: none;}
+}
+
+@media (max-width: 376px) {
+	.brand-info {
+		padding: 10px 45px 50px 45px;
+		height: 400px;
+	}
+
+	.brand-info:before {
+	  content: "";
+	  width: 110%;
+	  left: -5%;
+	}
+
+	.brand-info__image {
+		padding: 0;
+		height: 151px;
+		border: 4px;
+		flex: 0 1 auto;
+	}
+
+	.brand-info__text {padding: 0 0 20px 0;}
+
+	.brand-info__title {
+		line-height: 25px;
+		margin-bottom: 15px;
+	}
+
+	.brand-info__history {
+		font-size: 14px;
+		line-height: 17px;
+	}
+}
+/*MAIN <section class='stories'>*/
+.stories {margin: 0 0 82px 0;}
+
+.stories__maintitle {margin: 0 0 40px 0;}
+
+.stories-wrap {
+	display: flex;
+	justify-content: space-between;
+}
+
+.stories__item {width: 307px;}
+
+.stories__img {
+	height: 150px;
+	width: 100%;
+	margin: 0 0 30px 0;
+}
+
+.stories__title {
+	font-size: 18px;
+	line-height: 17px;
+	margin: 0 0 10px 0;
+}
+
+.stories__text {
+	font-size: 14px;
+	line-height: 18px;
+	margin: 0 0 16px 0;
+}
+
+.stories__details {
+	display: flex;
+	justify-content: space-between;
+	font-size: 12px;
+	line-height: 11px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ProjectMod2_EKh/pullRequest?prid=6247910)